src/IO/Importers/Tags.hpp

```diff
@@ -170,6 +173,25 @@ struct RegisteredElements : db::SimpleTag {
   using type = std::unordered_map<observers::ArrayComponentId, std::string>;
 };
 
+/// Indicates which volume data files have already been read.
+struct HasReadVolumeData : db::SimpleTag {
```

Comment:
What about `VolumeDataAlreadyRead`? I think the fun English `read` and `read` (present and past) being the same made me fumble when I read this the first time

Reply:
renamed